docs/core/tools/global-json.md

```diff
@@ -33,30 +35,103 @@ Note that this field:
 - Doesn't have globbing support, that is, the full version number has to be specified.
 - Doesn't support version ranges.
 
-The following example shows the contents of a *global.json* file:
+#### allowPrerelease
+
+- Type: `boolean`
+
+- Available since: .NET Core 3.0 SDK.
+
+Indicates whether the SDK resolver should consider prerelease versions when selecting the SDK version to use.
+
+If you don't set this value explicitly, the default value depends on whether you're running from Visual Studio:
+
+- If you're **not** in Visual Studio, the default value is `true`.
+- If you are in Visual Studio, it uses the prerelease status requested. That is, if you're using a Preview version of Visual Studio or you set the **Use previews of the .NET Core SDK** option (under **Tools** > **Options** > **Environment** > **Preview Features**), the default value is `true`; otherwise, `false`.
+
+#### rollForward
+
+- Type: `string`
+
+- Available since: .NET Core 3.0 SDK.
+
+The roll-forward policy to use when selecting an SDK version, either as a fallback when a specific SDK version is missing or as a directive to use a later version. A [version](#version) must be specified with a `rollForward` value, unless you're setting it to `latestMajor`. 
```

Comment:
We need to replace most use of "latest" with "highest". In the 2.x SDKs, this is not the same. Patches are latest.

```suggestion
The roll-forward policy to use when selecting an SDK version, either as a fallback when a specific SDK version is missing or as a directive to use a higher version. A [version](#version) must be specified with a `rollForward` value, unless you're setting it to `latestMajor`. 
```

Reply:
I agree with @tdykstra. a later version or a higher version are synonyms with a later version being more common in docs. 

Note that I'm not saying latest, I'm saying a later version, but I can change to higher if you think it's confusing.